Blockstate update for 1.14+

miclav opened the pull request:
Hi
Here is my PR for the 1.14 blockstates. It includes changes to accommodate:
- Bamboo
- Composter
- All the signs
I'm however not sure if for each one I took the right approach.
There's still the bell that doesn't display completely, only the frame is rendered at the moment.

miclav committed:
Bamboo fix

miclav committed:
Change sign processing to the multiple sign types of 1.14

miclav committed:
Fixing composter level 0

miclav committed:
Accomodate changes in 1.15 doublechest texture mapping

miclav commented:
Now with 1.15 update :)

miclav committed:
Adding Bell rendering

Previously only the support was rendered, now the bell is displayed too

miclav committed:
1,16 update to assets

- namespace have been added in some files, and should be removed to furthur process the asset
- Walls have now new properties that must be explicitly set.

elamperti commented:
@m0r13 This PR has changes in [`blockcrafter/mcmodel.py`](https://github.com/mapcrafter/blockcrafter/pull/7/files#diff-70a8a8ea7505cef2e64af8253894febe). I think everything is looking good but it'd be good to have your approval on this one

fredkilbourn commented:
@elamperti @m0r13 if you guys accept this can you please also be sure to update the published docker image?

elamperti commented:
@fredkilbourn I think build are not automated for this repo so we'll have to keep an eye on that. I'd like that to be fixed but I'll focus efforts on merging `world113` in the mapcrafter repo first to start moving through the gears :)

fredkilbourn commented:
No problem thanks!

bemk committed:
Fix biome colouring for stem blocks

This issue solves the green tints on:
* mushroom stems
* warped stems
* stripped warped stems
* crimson stems
* stripped crimson stems

miclav committed:
Merge pull request #1 from bemk/master

Fix biome colouring for stem blocks

miclav committed:
Cleanner glass, crispier textures and missing signs and conduit

miclav committed:
Remove hacked default variant and add a configurable default variant

+ Make the atlas wider to reduce the crazy height of the png
+ Clean up blockstates

miclav committed:
waterlogged lighting_rod + chain

bemk committed:
Apply a temporary fix to one of the dependencies

It's a dirty little hack to do this in the Dockerfile, but for now this
means the end of my yak shaving.

bemk committed:
The block grass_path was renamed to block_path in 1.17

miclav committed:
Merge pull request #2 from bemk/master

Apply a temporary fix to one of the dependencies and fix the grass_path block problem
